### ISSUE TITLE
Make install scripts more robust

### ISSUE THREAD

**Issue by spco** [AtChem/AtChem2#399]
Specifically, make them fail loudly, rather than silently as they do now - that helps with debugging, especially on Travis.

**Comment by rs028** [comment]
@spco, can you elaborate a little, what did you have in mind when you opened this issue? I may be able to look into that.

**Comment by spco** [comment]
As it stands, some of the scripts do error-catching, and some don't. Normally this is in the form
```
if [ $? -ne 0 ] ; then
  echo "some error or other"
  exit 1
fi
```
For example, in `install_openlibm.sh`, some parts are guarded with that clause, but the final `make -j` command is not. So if the `make -j` clause fails, the script carries on and still does `exit 0` (which means success). This can make debugging hard, particularly on GitHub Actions, because the stage looks like it's succeeded but has in fact failed, requiring you to dig directly into the logs to find the problem.

So I propose guarding each step in this way in each script. Currently it's somewhat inconsistent.

**Comment by rs028** [comment]
Okay I understand. I will have a go. Thanks!

**Comment by rs028** [comment]
This should be now solved by PR #496
